index.js

```diff
@@ -5,11 +5,13 @@
 var di = require('di'),
     _ = require('lodash'),
     core = require('on-core')(di);
+var temp = require('temp').track();
```

Comment:
Please remove ".track()". The temp module's track feature is only usable for shorter-run processes to delete all temp files _when the process exits_. In our case, that array of file names (already deleted or otherwise) will OOM the process/machine.